packages/editor-ui/src/components/UpdatesPanel.vue

```diff
@@ -7,21 +7,22 @@ import VersionCard from './VersionCard.vue';
 import { VERSIONS_MODAL_KEY } from '../constants';
 import { useVersionsStore } from '@/stores/versions.store';
 import { useI18n } from '@/composables/useI18n';
+import { useUIStore } from '@/stores/ui.store';
 
 const versionsStore = useVersionsStore();
+const uiStore = useUIStore();
+
 const i18n = useI18n();
 
-const nextVersions = computed(() => {
-	return versionsStore.nextVersions;
-});
+const nextVersions = computed(() => versionsStore.nextVersions);
+
+const currentVersion = computed(() => versionsStore.currentVersion);
 
-const currentVersion = computed(() => {
-	return versionsStore.currentVersion;
-});
+const infoUrl = computed(() => versionsStore.infoUrl);
 
-const infoUrl = computed(() => {
-	return versionsStore.infoUrl;
-});
+const onInfoUrlClick = async () => {
+	await uiStore.goToVersions();
+};
```

Comment:
nit - Maybe we don't need this then.

can just use the function directly below 
`@click="uiStore.goToVersions"`